iOS/Collection View Management /DiffableCollectionViewController.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  DiffableCollectionViewController.swift
+//  Jibber
+//
+//  Created by Benji Dodgson on 9/25/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+class newDataSource<SectionType: Hashable, ItemType: Hashable>: CollectionViewDataSource<SectionType, ItemType> {
+
+}
+
+class DiffableCollectionViewController<SectionType: Hashable, ItemType: Hashable, DataSource: newDataSource<SectionType, ItemType>>: ViewController, UICollectionViewDelegate {
```

Comment:
New DiffableCollectionViewController. I really like it and seems really flexible.

Reply:
Cool, so it's connecting the collection view and datasource together.